pyproject.toml

```diff
@@ -13,5 +13,7 @@ classifiers = [
 
 
 [tool.maturin]
+bindings = 'pyo3'
 features = ["pyo3/extension-module"]
-python-source = "pychiquito"
+python-source = "python"
+module-name = "chiquito.rust_chiquito"
```

Comment:
so this makes rust_chiquito a sub module of package chiquito?

Reply:
Right, an example using it [here](https://github.com/trangnv/chiquito/blob/65bb32dd651145ff294d03cc14b3548af9ff2ae9/python/chiquito/dsl.py#L4) 
In general, `chiquito.rust_chiquito` module has function listed [here](https://github.com/trangnv/chiquito/blob/65bb32dd651145ff294d03cc14b3548af9ff2ae9/src/lib.rs#L44)

Reply:
Thanks, that makes much more sense structure wise